### DISCUSSION TITLE
Drag n Drop

### DISCUSSION THREAD

**Discussion by ashmoreinc** [File-New-Project/EarTrumpet#619] [Completed ideas]
I think a drag and drop feature so you can drag sound sources from one output device too another would work really well with this app.

**Comment by TehMillhouse** [comment]
I came here to suggest this. I would love this. Especially in the current home office situation, I find myself shuffling audio streams back and forth between my speakers and my headphones all the time. Letting me drag audio streams between the device sections of the flyout would make this much easier and faster.

It'd also be neat to have the sound instantly move devices *while dragging*. I've often been in setups where I had multiple sound devices with similar names. Having the stream migrate while I'm still dragging would enable me to just drag the icon to where it sounds right.

I could see the "migrate while dragging" thing be risky in setups where one device is much louder than the others, so this would definitely be something I'd let people disable.

**Comment by kellpete** [comment]
I agree, but this looks like a duplicate of #662

**Reply by ashmoreinc** [reply]
So it is, thanks for pointing that out

**Comment by riverar** [comment]
Doing some spring cleaning 🌹 Moved the discussion to complete as this is a duplicate.

**Comment by ghost** [comment]
hey,
Can you add a feature Drag and drop application from audio device to another to change their playback device?
